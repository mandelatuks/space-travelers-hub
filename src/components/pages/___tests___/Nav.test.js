import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../../../Redux/Configurestore';
import Nav from '../Navbar';
import '@testing-library/jest-dom/extend-expect';

test('test the MissionPage snapshot', () => {
  const missionRenderer = renderer.create(
    <Provider store={store}>
    <BrowserRouter>
      <Nav />
      </BrowserRouter>
    </Provider>
    ,
  ).toJSON();
  expect(missionRenderer).toMatchSnapshot();
});