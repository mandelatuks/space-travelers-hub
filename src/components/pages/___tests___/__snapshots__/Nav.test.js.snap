// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the MissionPage snapshot 1`] = `
<header>
  <h1>
    <img
      alt="logo"
      className="app-logo"
      src="logo.png"
    />
     
    Space Traveler's Hub
  </h1>
  <nav
    className="navBar"
  >
    <ul>
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li>
        <a
          className=""
          href="/mission"
          onClick={[Function]}
        >
          Missions
        </a>
      </li>
      <li>
        |
      </li>
      <li>
        <a
          className=""
          href="/my-profile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
